### PR TITLE
Identity with link info

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/services-team 
https://wikia-inc.atlassian.net/browse/SERVICES-987

Also return information on whether or not the Google account associated with the access token is already linked to Wikia as part of the response to `/google/identity`. This will be useful when signing up with a Google access token

### CODE REVIEW

File: service/external-auth/src/main/java/com/wikia/services/externalauth/resources/GoogleAuthResource.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import com.wikia.pandora.http.StandardResponses;
 import com.wikia.services.externalauth.config.TokenCookieConfig;
 import com.wikia.services.externalauth.domain.AuthenticatedAccount;
```
Comment (WikiaTech) [inline service/external-auth/src/main/java/com/wikia/services/externalauth/resources/GoogleAuthResource.java]: MINOR Remove this unused import 'com.wikia.services.externalauth.domain.AuthenticatedAccount'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

File: service/external-auth/src/test/java/com/wikia/services/externalauth/resources/GoogleAuthResourceTest.java [hunk 2]
```diff
@@ -19,6 +22,7 @@
 import com.wikia.dropwizard.auth.APIGatewayAuthFactory;
 import com.wikia.services.externalauth.config.TokenCookieConfig;
 import com.wikia.services.externalauth.domain.AuthenticatedAccount;
```
Comment (WikiaTech) [inline service/external-auth/src/test/java/com/wikia/services/externalauth/resources/GoogleAuthResourceTest.java]: MINOR Remove this unused import 'com.wikia.services.externalauth.domain.AuthenticatedAccount'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by drsnyder** [comment]
lgtm

**Pull request merged**